pygmyui/pygmy/views.py

```diff
@@ -112,15 +112,25 @@ def short_link_stats(request, code):
     pygmy_client = pygmy_client_object(settings, request)
     if request.method == 'GET':
         try:
-            clickmeta = pygmy_client.link_stats(code)
+            clickmeta = pygmy_client.link_stats(code)   
             clickmeta['country_stats'] = sorted(
                 clickmeta['country_stats'].items(),
                 key=operator.itemgetter(1),
                 reverse=True)
+         
+            stats =[(country,iso2full.get(country,"unknown"),hits) for (country,
+                                                hits) in clickmeta['country_stats']]
+            clickmeta['country_stats'] = stats
+
             clickmeta['referrer'] = sorted(
                 clickmeta['referrer'].items(),
                 key=operator.itemgetter(1),
                 reverse=True)
+
+            refs = [(country,iso2full.get(country,"unknown"),hits) for (country,
```

Comment:
@whoami38 this part is not required as `clickmeta['referrer']` contains referrer info like `[('https://google.com', 7), ('https://gmail.com', 3)]` where index 0 is the referral source and index 1 is the referral counts.

Check the referrer section here https://pygy.co/pygmy+

Reply:
@amitt001 Sorry for misunderstanding, i have changed this part in view file, as well as, in the template.